users/models.py

```diff
@@ -0,0 +1,9 @@
+from django.utils.timezone import now
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class CustomUser(AbstractUser):
+    birthday = models.DateField('%m/%d/%y')
+    is_active = models.BooleanField(default=False)
```

Comment:
You probably don't need this field since you have an `activate_date` field. You can determine if user has activated the account by checking if `activate_date` is not `None`.